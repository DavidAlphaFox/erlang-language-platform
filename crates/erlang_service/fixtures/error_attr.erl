-module(error_attr).

-error("alamakota").

-error("two", "terms").

-error(NotATerm).
