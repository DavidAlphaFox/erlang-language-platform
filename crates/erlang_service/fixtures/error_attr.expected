AST
{attribute,{0,0},file,{"fixtures/error_attr.erl",0}}.
{attribute,{0,19},module,error_attr}.
{error,{{23,28},elp_epp,{error,"alamakota"}}}.
{error,{{45,50},elp_epp,{bad,error}}}.
{error,{{70,75},elp_epp,{bad,error}}}.
{eof,{87,87}}.


STUB
{attribute,{0,0},file,{"fixtures/error_attr.erl",0}}.
{attribute,{0,19},module,error_attr}.


WARNINGS
[]

ERRORS
[
    ParseError {
        path: "fixtures/error_attr.erl",
        location: Some(
            Normal(
                23..28,
            ),
        ),
        msg: "-error(\"alamakota\").",
        code: "E1522",
    },
    ParseError {
        path: "fixtures/error_attr.erl",
        location: Some(
            Normal(
                45..50,
            ),
        ),
        msg: "badly formed 'error'",
        code: "E1501",
    },
    ParseError {
        path: "fixtures/error_attr.erl",
        location: Some(
            Normal(
                70..75,
            ),
        ),
        msg: "badly formed 'error'",
        code: "E1501",
    },
]
